actions/release-notifications/index.js

```diff
@@ -0,0 +1,127 @@
+const core = require('@actions/core');
```

Comment:
One thing we need here actually is the copyright header added to the file

Reply:
Hi @tay1orjones : Yes. I've added it now. Please let me know in case of any changes needed. Thanks